src/sentry/integrations/slack/message_builder/notifications/rule_save_edit.py

```diff
@@ -40,13 +42,21 @@ def build(self) -> SlackBlock:
         rule_url = self.linkify_rule()
         project = self.rule.project.slug
         if self.new:
-            rule_text = f"{rule_url} was created in the *{project}* project and will send notifications here."
+            rule_text = f"Alert rule {rule_url} was created in the *{project}* project and will send notifications here."
         else:
-            rule_text = f"{rule_url} in the *{project}* project was updated."
+            rule_text = f"Alert rule {rule_url} in the *{project}* project was updated."
+            # TODO potentially use old name if it's changed?
 
-        # TODO add short summary of the trigger & filter conditions
         blocks.append(self.get_markdown_block(rule_text))
 
+        if not self.new and self.changed:
+            changes_text = "*Changes*\n"
+            for label, changes in self.changed.items():
```

Comment:
nit: It looks like we don't use the keys here so we can iterate through values only?